### ISSUE TITLE
Radio buttons in iDeal and CBC/KBC lists highlight the payment method before one is chosen

### ISSUE THREAD

**Issue by tilmanbremer** [mollie/gambio#32]
I updated the plugin today to 2.0.6 to fix issue #15. Gambio is now on 4.3.3.0.

It works now, so first of all, thanks a lot for the fix and the quick response! But, I have another issue, unfortunately. If the banks are displayed as radio buttons, the first will be selected automatically. That causes the list to expand and the payment method to appear colored, as if the payment method as a whole was chosen (in my case, that is the main shop color red). As CBC/KBC and iDEAL use the same list type, the last of the two that appears on the page will be expanded and highlighted red, in my case that is CBC/KBC.

This is how it looks:
![grafik](https://user-images.githubusercontent.com/20415107/107125258-fa6a5880-68a8-11eb-858f-58ceb9bf9aa4.png)

As a workaround, I set both payment methods back to lists for now.

**Comment by logeecom** [comment]
Hi @tilmanbremer,

The problem was with preselecting the first issuer from the list, so we have removed the preselection, and on form submit we have added check if the issuer is selected in case of the method which supports the issuer list is active. If no, the customer will not be able to proceed with the checkout and the error message will be displayed.

This fix will be available in the next plugin release next week.  

Best regards.
